circle.yml

```diff
@@ -0,0 +1,9 @@
+machine:
+  node:
+    version: 5.3.0
```

Comment:
Seems like a good idea to bring the project in line with the newer version, thanks for the heads up @ntwb! 👍